test_spec_PY3_GRETA

```diff
@@ -0,0 +1,9 @@
+# Excludes
```

Comment:
`test_spec_SKA3`.  This is the generic one that should pass on any Ska3 installation.